discovery-provider/src/tasks/celery_app.py

```diff
@@ -9,3 +17,74 @@
 # Celery removes all configured loggers. This setting prevents
 # Celery from overriding the configured logger set in create_celery()
 celery.conf.update({"worker_hijack_root_logger": False})
+
+
+def celery_worker(logger, db, redis, table_name, aggregate_func):
```

Comment:
Maybe make a more specific name like update_aggregate_table and move these functions to utils? In other cases, lock timeout may need to be passed in. Maybe we can get @dmanjunath to confirm if this is good to refactor.

Reply:
Yeah, I'm good to move it to utils. Let's decide on the best placement and I can move it there.

Indeed, figured we would need to customize more as we get more adoption of this pattern. Figured to leave that parameterization alone since I tried to only mess with aggregate_user and aggregate_tracks for this PR and both had the same lock timeout.

Reply:
agreed with @isaacsolo. similar to https://github.com/AudiusProject/audius-protocol/pull/2352/files#r794915492, if you want to generalize this, lets move this into a utils.py for index aggregate user/track/plays